docs/rector_rules_overview.md

```diff
@@ -1,4 +1,4 @@
-# 19 Rules Overview
+# 20 Rules Overview
```

Comment:
please rebase and revert it, we use website as doc source now 

https://getrector.com/find-rule?rectorSet=doctrine-doctrineorm-214